src/lib/tableland-calls.ts

```diff
@@ -16,6 +16,9 @@ export interface RpcParams {
   statement?: string;
   /* eslint-disable-next-line camelcase */
   txn_hash?: string;
+  caller?: string;
```

Comment:
This whole interface is starting to get bloated. What about separating out the various RpcParam alternatives into separate ones, and then having the method just accept a union type? So something like:

```
interface SetControllerParams {
  controller: string;
  caller: string;
  token_id: string;
}

interface FooParams {}

interface BarParams {}

... // various combinations...

export type RpcParams = SetControllerParams | FooParams | BarParams;
```

That way, we'd actually get type checking for the various methods! Because Typescript is smart enough to know that if you have `caller` and `token_id`, then it _must_ be `SetControllerParams`, which means you must also have a `controller` property! Nice!